syz-ci/jobs.go

```diff
@@ -523,6 +523,9 @@ func (jp *JobProcessor) bisect(job *Job, mgrcfg *mgrconfig.Config) error {
 	res, err := bisect.Run(cfg)
 	resp.Log = trace.Bytes()
 	if err != nil {
+		if _, ok := err.(*bisect.InfraError); ok {
+			resp.Flags |= dashapi.BisectResultInfraError
```

Comment:
What would be an example of an error that is not infra error?

Reply:
For example, if a bisection takes more than 8 hours

Reply:
One other case when we probably don't want to redo the bisection: `the crash wasn't reproduced on the original commit`